### PR TITLE
moved inline styles from ejs files to styles.css issue#553

### PR DESCRIPTION
**Pull request by Ayush0730**
## What is the change?
I have removed all css from ejs files and replaced some of them with bootstrap classes, and others with different classes defined in styles.css

## Related issue?
close: #553 

## How was it tested?
After replacing some classes with bootstrap classes, I have checked the visual appearance and it looks same to me (very minor difference in margin etc.)

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

### CODE REVIEW

File: views/404.ejs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <%- include("partials/header", {isAuthenticated}) -%>
 <div class="container-centre content-body-font">
     <img src="/images/404.png">
-    <a href='https://www.freepik.com/vectors/website' style="display: none;">Website vector created by stories - www.freepik.com</a>
+    <a href='https://www.freepik.com/vectors/website' class="d-none">Website vector created by stories - www.freepik.com</a>
```
Comment (Pranav016) [inline views/404.ejs:4]: I didn't notice this tag earlier, thanks for bringing it into notice.
 you can remove this tag and remove its corresponding css

File: public/css/styles.css [hunk 2]
```diff
@@ -224,6 +224,18 @@ a::after {
...
+}
+
+.f-s-2-4rem {
+    font-size: 2.4rem;
+}
+
+.fw-600 {
+    font-weight: 600;
+}
+
```
Comment (Pranav016) [inline public/css/styles.css:238]: rename these class names, could look confusing for a new contributors when he opens the ejs file and notices these classes

File: public/css/styles.css [hunk 3]
```diff
@@ -648,3 +665,51 @@ input[name='perPage'] {
...
+.footer-heading {
+    color: rgb(255, 231, 10);
+}
+
+.social-icon {
+    font-size: 2rem;
+    margin-right: 1em;
+}
+
+.facebook {
```
Comment (Pranav016) [inline public/css/styles.css:697]: add comments throughout the css file that helps distinguishing which section's css begins where..

**Review by Pranav016 — CHANGES_REQUESTED** [review]
PR looks good, just a few changes

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level3/

File: public/css/styles.css (hunk 2 above)
Comment (Ayush0730) [reply public/css/styles.css:238]: Okay. What about fsize and fweight? Are these names OK?

**Comment by Ayush0730** [comment]
@Pranav016 I have done the requested changes. Please check if it is okay

**Review by Pranav016 — APPROVED** [review]
Class names could be better but looks fine for now..

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level3/

**Pull request merged**